$NetBSD: patch-ac,v 1.5 2015/05/12 07:49:16 kim Exp $

Fix bashisms. Only doing configure, to avoid regen from configure.in.

--- configure.orig	2015-04-08 09:30:42.000000000 +0000
+++ configure	2015-05-12 07:01:52.000000000 +0000
@@ -13435,7 +13435,7 @@
   have_kate=no
 fi
 
-if test "x$have_kate" == "xyes"
+if test "x$have_kate" = "xyes"
 then
   { $as_echo "$as_me:${as_lineno-$LINENO}: checking for kate_ogg_decode_headerin in -loggkate" >&5
 $as_echo_n "checking for kate_ogg_decode_headerin in -loggkate... " >&6; }
@@ -13479,7 +13479,7 @@
   have_kate=no
 fi
 
-  if test "x$have_kate" == "xyes"
+  if test "x$have_kate" = "xyes"
   then
     KATE_LIBS="-loggkate -lkate -logg"
 
